packages/node/src/handlers.ts

```diff
@@ -370,6 +371,20 @@ export type RequestHandlerOptions = ParseRequestOptions & {
 export function requestHandler(
   options?: RequestHandlerOptions,
 ): (req: http.IncomingMessage, res: http.ServerResponse, next: (error?: any) => void) => void {
+  const currentHub = getCurrentHub();
+  const client = currentHub.getClient<NodeClient>();
+  // Initialise an instance of SessionFlusher on the client when `autoSessionTracking` is enabled and the
+  // `requestHandler` middleware is used indicating that we are running in SessionAggregates mode
+  if (client && isAutoSessionTrackingEnabled(client)) {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    (client as any)._initSessionFlusher();
```

Comment:
I really don't like the fact that we have so many private properties/methods accesses and so many `any` casts :(